src/main/AndroidManifest.xml

```diff
@@ -63,6 +63,17 @@
       android:screenOrientation="portrait"
       android:configChanges="orientation|screenSize"
       android:launchMode="singleTop"/>
+
+    <provider
```

Comment:
Allows the app to create files in its directory.

Reply:
To take a picture, we need to create a file to store the pic. 
To record video, we don't need to create a file 🤷‍♀️